internal/client/datadog/formatter.go

```diff
@@ -27,8 +27,14 @@ func (f Formatter) Format(prefix string, maxPostBytes uint32, data map[metric.Me
 		f.log.Errorf("Error formatting metrics payload: %v", err)
 		return result
 	}
-	if uint32(len(compressedSeriesBytes)) > maxPostBytes && canSplit(data) {
-		metricsA, metricsB := splitPoints(data)
+
+	if uint32(len(compressedSeriesBytes)) > maxPostBytes {
+		if len(data) == 1 {
+			return nil
```

Comment:
Let logs something here to notify that we did drop something. We even have access to the metric name so that's great information to log too